torch/package/package_importer.py

```diff
@@ -317,6 +343,36 @@ def _make_module(
         ns["__builtins__"] = self.patched_builtins
         ns["__torch_package__"] = True
 
+        node = self._get_node_from_name(name)
+        if isinstance(node, _SelectiveExternNode):
+
+            ns["_extern_copy"] = importlib.import_module(name)
+
+            def get_selectively_interned_module_names(cur_node, name):
+                if isinstance(cur_node, _ExternNode):
```

Comment:
for my understanding: can `_ExternNode` ever be a child of a `_SelectiveExternNode`? I thought no?

Reply:
It should normally be the case no? Say we want to import torch.nn which is always externed. In that case torch is a _SelectiveExternNode node and nn is a _ExternNode. Otherwise, we don't expose these modules to the exporter.

Reply:
My understanding is that currently, the a `_SelectiveInternNode`'s children refer ONLY to the interned modules. If a module is not in the children, only then will we use the external importer. I'm basing this off L429 in this file. That suggests to me there will never be an `_ExternNode` child of a `_SelectiveInternNode`, right?

Reply:
Oh gotcha, I think I got confused because Externed modules under `_SelectiveInternNode`s are not in the externed list. Will fix that now.